### DISCUSSION TITLE
Where can I find example code for XGboost multiclass classification with Optuna?

### DISCUSSION THREAD

**Discussion by keyuchen21** [optuna/optuna#3981] [Q&A]
### What is an issue?

All current example are for binary classification with Xgboost example code
https://github.com/optuna/optuna-examples/blob/main/xgboost/xgboost_simple.py

Any one could help me out?

**Comment by nzw0301** [comment]
I suggest asking the question on the Xgboost community rather than here to get the answer quickly because it sounds unrelated to Optuna.

**Reply by keyuchen21** [reply]
what i mean is Optuna's objective function towards XGboost multi-class classification

**Reply by nzw0301** [reply]
I suppose we use a loss for training and a metric as an objective for multi-class classification problems in xgboost way. Again, this sounds more xbgoost problem rather than optuna...
